File: server/src/main/java/com/example/schoolsite/workWithDatabase/repo/CalendarRepository.java
package com.example.schoolsite.workWithDatabase.repo;

import com.example.schoolsite.entity.Calendar;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CalendarRepository extends JpaRepository<Calendar, Long> {
    Calendar findByLessonNumberAndWeekDay(int lessonNumber, int weekDay);
}

File: server/src/main/java/com/example/schoolsite/workWithDatabase/repo/CalendarRepository.kt
package com.example.schoolsite.workWithDatabase.repo

import com.example.schoolsite.entity.Calendar
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CalendarRepository : JpaRepository<Calendar?, Long?> {
    fun findByLessonNumberAndWeekDay(lessonNumber: Int, weekDay: Int): Calendar?
}